.github/workflows/ibis-backends.yml

```diff
@@ -130,6 +130,7 @@ jobs:
         python-version:
           - "3.7"
           - "3.9"
+        deps: null
```

Comment:
So we should have three variations here, one for 3.9 and two for 3.9?

Reply:
I guess you mean by the second 3.9 3.7 :)

3.7 runs with the lower bounds and unrestricted bounds, and 3.9 with just the unrestricted